src/main/java/org/humancellatlas/ingest/project/web/ProjectController.java

```diff
@@ -85,7 +85,7 @@
 
         List<String> allowedFields = List.of("content", "releaseDate",
                 "primaryWrangler", "secondaryWrangler", "wranglingState", "wranglingPriority", "wranglingNotes",
-                "accessionDate", "technology", "organ", "dataAccess", "identifyingOrganisms", "validationErrors", "isInCatalogue", "publicationsInfo");
+                "accessionDate", "technology", "organ", "cellCount", "dataAccess", "identifyingOrganisms", "validationErrors", "isInCatalogue", "publicationsInfo");
```

Comment:
I would format this list as one entry per line. Even sort it. It would make diffs easier to notice.